src/amo/components/SearchForm/index.js

```diff
@@ -58,13 +58,13 @@ export class SearchFormBase extends React.Component<Props> {
     };
 
     let openSearchTitle = i18n.sprintf(
```

Comment:
We do not need the call to `i18n.sprintf()` anymore so it can be removed here and below too. We also don't need the `i18nValues` variable.

Maybe we can have something like that in the end:

```
const openSearchTitle = clientApp === CLIENT_APP_ANDROID ? i18n.gettext(...) : i18n.gettext(...);
```